index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
                             <li><a href="#"><img src="img/github.jpg" alt="github icon"></a></li>
                             <li><a href="#"><img src="img/angellist.jpg" alt="angellist icon"></a></li>
                         </ul>
-        </section>  
+        </section>
+    </section>  
+            <!-- the first card............... -->
+            <section id="work-section">
+                <section class="cards">
+                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <div class="align-text">
+                    <h2 class="h2-size">Tonic</h2>
+                    <ul class="first-list">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                    <ul class="blue-list">
+                        <li>HTML</li>
+                        <li>Css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="work-btn btn">see project</button>
+                    </div>
+                    </section>
```

Comment:
Please note that only the main content should be a `section` tag, I suggest you use the `div` tag instead of the `section` tag for the cards. kindly make sure you fix them to follow best practices and HTML good semantics. 😸